sdk/search/azure-search-documents/azure/search/documents/aio/_search_client_async.py

```diff
@@ -117,7 +125,9 @@ async def get_document_count(self, **kwargs: Any) -> int:
         return int(await self._client.documents.count(**kwargs))
 
     @distributed_trace_async
-    async def get_document(self, key: str, selected_fields: Optional[List[str]] = None, **kwargs: Any) -> Dict:
+    async def get_document(
```

Comment:
Curious what caused all the formatting changes in this file. Seems counter to the black-opinionated formatting.

Reply:
Yes. Black has different preferences at different times. :)